ETL/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ if (CMAKE_USE_PTHREADS_INIT)
     set(HAVE_LIBPTHREAD ON)
 endif()
 
-# TODO: check for it
-set(HAVE_VASPRINTF ON)
+include(CheckCXXSymbolExists)
+check_cxx_symbol_exists(vsnprintf stdio.h HAVE_VSNPRINTF)
```

Comment:
vsnprintf must exist in any c++11 compiler by standard, so i've removed any checks from CMakeLists.txt and configure.ac